src/main/java/bio/terra/service/filedata/azure/blobstore/AzureBlobStorePdao.java

```diff
@@ -63,6 +65,16 @@ public AzureBlobStorePdao(
     this.azureAuthService = azureAuthService;
   }
 
+  private RequestRetryOptions getRetryOptions() {
+    return new RequestRetryOptions(
+        RetryPolicyType.EXPONENTIAL,
+        this.resourceConfiguration.getMaxRetries(),
+        this.resourceConfiguration.getRetryTimeoutSeconds(),
+        null,
+        null,
+        null);
```

Comment:
This is small, but smells weird. Could we provide an overloaded constructor that does this null-passing for us.

Reply:
Yeah it's a bit confusing...the null parameters here will use the azure-provided defaults for RequestRetryOptions. I'll add that change into https://github.com/DataBiosphere/jade-data-repo/pull/1027 so I can merge this in :)